Marlin/src/inc/Conditionals_adv.h

```diff
@@ -1240,7 +1240,7 @@
 #if LCD_IS_SERIAL_HOST && !defined(LCD_SERIAL_PORT)
   #if MB(BTT_SKR_MINI_E3_V1_0, BTT_SKR_MINI_E3_V1_2, BTT_SKR_MINI_E3_V2_0, BTT_SKR_MINI_E3_V3_0, BTT_SKR_E3_TURBO, BTT_OCTOPUS_V1_1)
     #define LCD_SERIAL_PORT 1
-  #elif MB(CREALITY_V24S1_301, CREALITY_V24S1_301F4, CREALITY_V423, MKS_ROBIN, PANOWIN_CUTLASS, KODAMA_BARDO)
+  #elif MB(CREALITY_V24S1_301, CREALITY_V24S1_301F4, BOARD_CREALITY_F401RE, CREALITY_V423, MKS_ROBIN, PANOWIN_CUTLASS, KODAMA_BARDO)
```

Comment:
You’ll need to drop the `BOARD_` portion of the define since that is already handled with the `MB()` macro.

Reply:
ouch i'm blind thanks